modules/ebi-metagenomics/hhsuite/buildhhdb/meta.yml

```diff
@@ -0,0 +1,47 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hhsuite_buildhhdb"
+description: create an HH database to use for hhblits searches
+keywords:
+  - HH
+  - model
+  - database
+tools:
+  - "hhsuite":
+      description: "HH-suite3 for fast remote homology detection and deep protein annotation"
+      homepage: "https://github.com/soedinglab/hh-suite"
+      documentation: "https://github.com/soedinglab/hh-suite/wiki"
+      tool_dev_url: "https://github.com/soedinglab/hh-suite"
+      doi: "10.1371/journal.pone.0082138"
+      licence: ["GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - a3m:
+      type: directory
+      description: Directory containing a3m format MSAs
+      pattern: "*"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - hh_db:
+      type: directory
+      description: HH db directory containing all precomputed required data and index files (HMMs and context states for prefiltering)
+      pattern: "*"
```

Comment:
do you need a pattern for a directory? I don't actually know haha

Reply:
You can find all choices in nf-core modules, so I don't think there is a consensus. Better remove it